docs/ibc/light-clients/updates-and-misbehaviour.md

```diff
@@ -0,0 +1,38 @@
+<!--
+order: 5
+-->
+
+# Handling Updates and Misbehaviour
```

Comment:
could we add pseudocode from the `UpdateClient` function which shows the usage of these 4 functions?

Something like:
```go
    if err := clientState.VerifyClientMessage(clientMessage); err != nil {
        return err
    }
    
    foundMisbehaviour := clientState.CheckForMisbehaviour(clientMessage)
    if foundMisbehaviour {
        clientState.UpdateStateOnMisbehaviour(clientMessage)
        // emit misbehaviour event
        return 
    }
    
    clientState.UpdateState(clientMessage) // expects no-op on duplicate header
    // emit update event
    return
}
```